app/controllers/kudos_controller.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+class KudosController < ApplicationController
+  before_action :set_kudos, only: %i[show edit update destroy]
+  before_action :authenticate_employee!, except: %i[index show]
+  before_action :correct_employee, only: %i[edit update destroy]
+
+  def index
+    render :index, locals: { kudos: Kudo.all }
+    #  @kudos = Kudo.all
+  end
+
+  def show
+    render :show, locals: { kudo: kudo }
+  end
```

Comment:
Acceptance Criteria's are mentioning only index, and new/create action - there is nothing about `edit`, and `destroy`, or am I missing something?

Reply:
no, you are right i deleted unnecessary methods :)